__tests__/get.test.js

```diff
@@ -61,6 +61,145 @@ describe("GET /api/reviews", () => {
         });
       });
   });
+  describe("queries", () => {
+    test("categories", () => {
+      return request(app)
+        .get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then((result) => {
+          expect(result.body.reviews.length).toBeGreaterThan(0);
+          result.body.reviews.forEach((reviews) => {
+            expect(reviews).toMatchObject({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: "dexterity",
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+        });
+    });
+    test("200 category exists but has no reviews associated", () => {
+      return request(app)
+        .get("/api/reviews?category=children's games")
+        .expect(200)
+        .then((result) => {
+          expect(result.body).toEqual([]);
+        });
+    });
+
+    describe("sort_by", () => {
+      test("shoudl default to created_at", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=default")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews.length).toBeGreaterThan(0);
+            expect(result.body.reviews[0]).toMatchObject({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: 7,
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+      });
+      test("sort_by", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=owner")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews.length).toBeGreaterThan(0);
+            expect(result.body.reviews[0]).toMatchObject({
```

Comment:
Ideally, jest-sorted should be used here, as you're just validating the first result, not all of them.